libs/common-transfers/src/lib/components/excluded-dates.component.ts

```diff
@@ -49,33 +43,41 @@ export class ExcludedDatesComponent implements PaymentFormField, OnInit, OnDestr
       this.config,
     ) as FormControl;
 
-    this.control.setValidators([
```

Comment:
- The future/past date validators are not needed (see other comment)
- The required one should not be added here, so end-user can set that from the omni payment config
- The excluded date validator moves to async